cmake/CMakeLists.txt

```diff
@@ -704,7 +701,14 @@ else()
   message(STATUS "Static library flags:    ${CMAKE_STATIC_LINKER_FLAGS}")
 endif()
 if(BUILD_MPI)
+  # get MPI compile settings from MPI4WIN imported target when cross-compiling
+  if((CMAKE_SYSTEM_NAME STREQUAL Windows) AND CMAKE_CROSSCOMPILING)
```

Comment:
I would just set these 3 variables inside `MPI4WIN.cmake` to keep hackery more localized.

Reply:
done